runtime/src/in_mem_accounts_index.rs

```diff
@@ -419,6 +419,11 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         self.storage.wait_dirty_or_aged.notify_one();
     }
 
+    fn should_remove_from_mem(&self, current_age: Age, entry: &AccountMapEntry<T>) -> bool {
+        // this could be tunable dynamically based on memory pressure
+        current_age == entry.age()
+    }
+
     fn flush_internal(&self) {
         let was_dirty = self.bin_dirty.swap(false, Ordering::Acquire);
```

Comment:
Not related to this PR, but this swap can probably be Relaxed. No need to change it now, just wanted to put it out there for us to be aware of and think about.

Reply:
can you tell me your thinking?

Reply:
also in description:
> There is a placeholder here for removing items from in-mem after writing to disk. It has complexity, so I broke it into a separate pr.

Reply:
Sorry, I think I'm wrong... I must've gotten mixed up with the `v.dirty()` and `v.set_dirty()` later down in the loop...